### PR TITLE
docs: add info about tutor-contrib-wordpress plugin

### PR DESCRIPTION
**Pull request by CodeWithEmad**
This will introduce the `tutor-contrib-wordpress` plugin for `tutor` inside documents. also, clean up some tailing whitespace and reformat an XML file.

### CODE REVIEW

**Comment by openedx-webhooks** [comment]
Thanks for the pull request, @CodeWithEmad!

### What's next?

*Please work through the following steps to get your changes ready for engineering review:*

#### :radio_button: Get product approval

If you haven't already, [check this list](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Does-my-contribution-require-Product-Review%3F) to see if your contribution needs to go through the product review process.

- If it does, you'll need to submit a product proposal for your contribution, and have it reviewed by the [Product Working Group](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3449028609/Product+Working+Group).
    - This process (including the steps you'll need to take) is documented [here](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3875962884/How+to+submit+an+open+source+contribution+for+Product+Review#Product-Review-Process).
- If it doesn't, simply proceed with the next step.

#### :radio_button: Provide context

To help your reviewers and other members of the community understand the purpose and larger context of your changes, feel free to add as much of the following information to the PR description as you can:

- Dependencies
  > This PR must be merged before / after / at the same time as ...
- Blockers
  > This PR is waiting for OEP-1234 to be accepted.
- Timeline information
  > This PR must be merged by XX date **because** ...
- Partner information
  > This is for a course on edx.org.
- Supporting documentation
- Relevant [Open edX discussion forum](https://discuss.openedx.org/) threads





#### :radio_button: Get a green build

If one or more checks are failing, continue working on your changes until this is no longer the case and your build turns green.



#### :radio_button: Let us know that your PR is ready for review:

### Who will review my changes?


This repository is currently maintained by `@mafermazu`.  Tag them in a comment and let them know that your changes are ready for review.


### Where can I find more information?

If you'd like to get more details on all aspects of the review process for open source pull requests (OSPRs), check out the following resources:

- [Overview of Review Process for Community Contributions](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/FAQ-about-pull-requests.html)
- [Pull Request Status Guide](https://docs.openedx.org/en/latest/developers/references/developer_guide/process/pull-request-statuses.html)
- [Making changes to your pull request](https://docs.openedx.org/en/latest/documentors/how-tos/make_changes_to_your_pull_request.html)

### When can I expect my changes to be merged?

Our goal is to get community contributions seen and reviewed as efficiently as possible.

However, the amount of time that it takes to review and merge a PR can vary significantly based on factors such as:

- The size and impact of the changes that it introduces
- The need for product review
- Maintenance status of the parent repository

:bulb: *As a result it may take up to several weeks or months to complete a review and merge your PR.*

**Comment by MaferMazu** [comment]
Hello @CodeWithEmad, thanks for your patience. I'll take a look at this PR this week.

**Comment by MaferMazu** [comment]
Sorry I couldn't review it before. I will prioritize this in the following days.

**Comment by MaferMazu** [comment]
Hello @CodeWithEmad, everything looks good so far. I will test the `tutor-contrib-wordpress`, and with that, we can merge this 🙌
Thanks for the contribution.

**Comment by MaferMazu** [comment]
Yes, as I said, the fix and the additional information you added look good. However, I needed clarification about the plugin.

In my understanding, the `tutor-contrib-wordpress` allows you to:
- Print the application info needed to integrate the WordPress and the Open edX platform with `tutor wordpress config`.
- Create a user, and two applications could be used to fill the configuration here: https://docs.openedx.org/projects/wordpress-ecommerce-plugin/en/latest/plugin_quickstart.html#configure-the-plugin

Are they the two functions the `tutor-contrib-wordpress` plugin has? Please correct me if I am wrong.

I ask this because when I read `tutor-contrib-wordpress`, I expected to have a plugin that would launch a WordPress site when I launched the platform. Something like [tutor-contrib-codejail](https://github.com/eduNEXT/tutor-contrib-codejail) or [tutor-notes](https://github.com/overhangio/tutor-notes). And I think the readme description is not so clear with that:

> This is a [Tutor](https://docs.tutor.edly.io/) plugin for integrating the [Open edX Commerce WordPress Plugin](https://github.com/openedx/openedx-wordpress-ecommerce) with your Open edX instance. The plugin allows seamless synchronization and interaction between your WordPress site and the Open edX platform, providing a cohesive e-commerce experience. If you don't know how to install the WordPress plugin, check out this [installation guide](https://docs.openedx.org/projects/wordpress-ecommerce-plugin/en/latest/plugin_quickstart.html).

And with the info we are adding to the readme in this PR, I think we need a more accurate description to avoid misunderstood and manage expectations.

So, for this PR, I recommend being more direct and clear about what people can do with the plugin.
And for the plugin, I will open some issues/doubts about it in the plugin repo.

Please let me know what you think.

**Comment by CodeWithEmad** [comment]
Sorry for the late response, @MaferMazu. 

> Are they the two functions the tutor-contrib-wordpress plugin has?

Yes, using `tutor dev|local|k8s do init --limit=wordpress` or `tutor dev|local|k8s launch` after enabling the plugin will create the applications, and afterward, you can run `tutor wordpress config` to see the data used to create the applications and use them [here](https://docs.openedx.org/projects/wordpress-ecommerce-plugin/en/latest/plugin_quickstart.html#configure-the-plugin).


> I think we need a more accurate description to avoid misunderstood and manage expectations.

You're absolutely right. Having a note, mentioning that "this won't apply a WordPress instance" is necessary. However, now that you mentioned it, we can add a feature to have a WordPress instance in `dev|local|k8s` in the future.

**Comment by mphilbrick211** [comment]
Hi @CodeWithEmad @MaferMazu! Just checking in on this :)

**Comment by CodeWithEmad** [comment]
Hey @mphilbrick211. I've made the changes related to the "Accurate Description" in my repo's readme and made plans for the second part which is deploying a WordPress on local/k8s for the future which comes in a separate PR.

File: docs/source/how-tos/create_an_openedx_app.rst [hunk 1]
```diff
@@ -3,6 +3,8 @@ Create an Open edX Application for the Plugin Settings
 
 You will learn how to create an Open edX Application for filling out the form in the Open edX Sync Plugin Settings in our WordPress Settings.
 
+If you are using `Tutor`_ for managing your Open edX instance, you can install `tutor-contrib-wordpress`_ and use its commands to simplify the process.
```
Comment (MaferMazu) [inline docs/source/how-tos/create_an_openedx_app.rst:6]:
```suggestion
If you are using `Tutor`_ for managing your Open edX instance, you can install `tutor-contrib-wordpress`_ and use its commands to assist with the configuration and integration of the Open edX platform with an existing WordPress site.
```

I copied that from the plugin readme. https://github.com/CodeWithEmad/tutor-contrib-wordpress?tab=readme-ov-file#before-you-start

**Review by MaferMazu — COMMENTED** [review]
Thanks for your patience.
I have one comment left, and then we can merge this 🙌

**Comment by MaferMazu** [comment]
Sorry to add another feedback, but can we maintain the ending new line in the `phpcs.xml` file? I added the why in the reference.

Also, can we update the PR title to use docs as a commit type? Because this is not a feature, it is docs and chore/style, but we can prioritize the docs update. - This is no blocker comment; I can edit this in the merge, but it is for maintaining consistency.

Ref:
https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline

**Comment by CodeWithEmad** [comment]
You're absolutely right about the end of the file's new line. the formatter removed it and I didn't notice. I'll fix it.

**Review by MaferMazu — APPROVED** [review]
Thanks for this ✨

**Pull request merged**